homeassistant/components/frontend/__init__.py

```diff
@@ -277,15 +277,19 @@ def async_setup(hass, config):
     if is_dev:
         hass.http.register_static_path("/home-assistant-polymer", repo_path)
         sw_path = os.path.join(repo_path, "build/service_worker.js")
+        trans_path = os.path.join(repo_path, "build/translations")
         static_path = os.path.join(repo_path, 'hass_frontend')
 
     else:
         sw_path = os.path.join(frontend_path, "service_worker.js")
+        trans_path = os.path.join(frontend_path, "translations")
         static_path = frontend_path
 
     hass.http.register_static_path("/service_worker.js", sw_path, False)
     hass.http.register_static_path("/robots.txt",
                                    os.path.join(frontend_path, "robots.txt"))
+    hass.http.register_static_path("/static/translations",
```

Comment:
This is not necessary, static is already mounted and the transations are copied over: https://github.com/home-assistant/home-assistant-polymer/blob/master/script/build_frontend#L25-L26

Reply:
Oh I see now, it's overriding the /static/translations route in dev mode. We should only do this mount in dev mode. Probably also want to add a test to make sure that this route is hit before the generic `/static` catch all route